packages/popover/test/__snapshots__/Popover.js.snap

```diff
@@ -132,8 +132,9 @@ exports[`Popover UI Positioning Bottom 1`] = `
   aria-describedby="dialog-description-1"
   aria-label="Top PopOver"
   class="c0"
+  data-placement="bottom"
   role="dialog"
-  style="position: absolute; top: 0px; left: 0px; opacity: 0; pointer-events: none;"
+  style="position: fixed; top: 0px; left: 0px; transform: translate3d(NaNpx, NaNpx, 0); will-change: transform;"
```

Comment:
I'm not sure, but the state of the snapshot with this PR is the same as the state of the working snapshots before the breaking change.

If you look at the [history](https://github.com/priceline/design-system/commits/master/packages/popover/test/__snapshots__/Popover.js.snap) of the snapshot, the first two show the same `NaNpx` values. The last one, [ecosystem upgrades](https://github.com/priceline/design-system/commit/fade1644002a450db623069b3bcd9c5d3d0d2c03#diff-d5732357e972e7ff921ce1f89a8fb406), is the one that upgraded styled components and broke the popover, and that is what this current diff is against.

I'll do a little digging and see if I can find why it is happening

Reply:
@sdalonzo, maybe we could colla. and quickly double checking this snapshot observation in the [ecosystem upgrades](https://github.com/priceline/design-system/commit/fade1644002a450db623069b3bcd9c5d3d0d2c03#diff-d5732357e972e7ff921ce1f89a8fb406) to get to the bottom of the understanding to clarify, in interests of moving forward?

Reply:
Found this [related issue](https://github.com/FezVrasta/react-popper/issues/238) on react-popper, tried the proposed fix (wrapping Popper children in a div), no change. There is also an [open issue](https://github.com/FezVrasta/popper.js/issues/796) with the same problem on popper.js

Reply:
This might only be an issue with the snapshots. If you put in some curly braces here and add a console.log(style) before returning the content, and then open a Popover in the storybook, you get correct values like {position: "fixed", transform: "translate3d(5px, 320px, 0)", top: 0, left: 0, willChange: "transform"}.

Reply:
thank you for the research. i'm good with moving forward.